.cirrus.yml

```diff
@@ -211,11 +194,10 @@ build_task:
 validate_task:
     name: "Validate $DISTRO_NV Build"
     alias: validate
-    skip: *tags
+    skip: *branches_and_tags
```

Comment:
This is way above my ability to review. I've spent too long on it already, and I *think* it's okay, but IMO it's a pretty big deal to disable `make validate` on branch actions (whatever those are). I would like double- and triple-checking from @containers/podman-maintainers . 

Team, the question is, can `make validate` safely be skipped in these CI circumstances? As a reminder, that includes: 

 https://github.com/containers/podman/blob/efcd48b9668612eb975a299b79e7b6673a5addce/Makefile#L651

@cevich if you wouldn't mind re-explaining what `branches` means, I think many of our non-CI-wizard members (myself included) would appreciate it.

Reply:
Correction: This will be skipped on branch-push (i.e. PR merge), creating a new branch,  and on tag-push.

Esp. for tag-push, we want automation to be as simple as possible, for improved automation reliability.   Any non-essential task failures that require code changes to correct...would necessarily need a new tag to be created (potentially repeating most of the release process).  

For a new branch and a branch update...I'm not sure how to distinguish between the two in a 'skip' statement, so they're lumped together.

IMHO, the danger here is caused by nonsequential merging of PRs.  In other words, PRs that pass `validate` individually, but may not if merged in a different order.  However, in this case, I believe we can rely on either the `build` task failing (i.e. in the daily cirrus-cron build) or a future PR (based on the branch) catching the problem - since `validate` would run in that context.

Reply:
(@edsantiago I've just added a comment covering this in `.cirrus.yml`)

Reply:
Thanks. I guess it SGTM but I'd still like confirmatory review from @containers/podman-maintainers 

All else LGTM. Thanks.

Reply:
nod